src/FsAutoComplete.Core/SimplifyNameAnalyzer.fs

```diff
@@ -17,13 +17,18 @@ module SimplifyNameDiagnosticAnalyzer =
                 symbolUses
                 |> Array.filter (fun symbolUse -> not symbolUse.IsFromOpenStatement)
                 |> Array.Parallel.map (fun symbolUse ->
+
                     let lineStr = source.[symbolUse.RangeAlternate.StartLine - 1].ToString()
-                    // for `System.DateTime.Now` it returns ([|"System"; "DateTime"|], "Now")
-                    let partialName = QuickParse.GetPartialLongNameEx(lineStr, symbolUse.RangeAlternate.EndColumn - 1)
-                    // `symbolUse.RangeAlternate.Start` does not point to the start of plid, it points to start of `name`,
-                    // so we have to calculate plid's start ourselves.
-                    let plidStartCol = symbolUse.RangeAlternate.EndColumn - partialName.PartialIdent.Length - (getPlidLength partialName.QualifyingIdents)
-                    symbolUse, partialName.QualifyingIdents, plidStartCol, partialName.PartialIdent)
+                    if symbolUse.RangeAlternate.EndLine <> symbolUse.RangeAlternate.StartLine then
+                        // Problem: EndColumn > lineStr.Length -> logic needs to account for multiple lines..
+                        symbolUse, [], 0, ""
```

Comment:
There seems to be a logic flaw here (something changed with the merge as there was a conflict, but I think this is still true)

Reply:
I'm not sure if I understand why this needs to be changed. How can symbol **usage** be longer than one line? Can you post some sample code for that?

Reply:
The code in question was [this](https://github.com/fsharp/FAKE/blob/197a3e5964ef45e7026bf480f2fc0bb8af8d5c4c/build.fsx#L349-L352)
```fsharp
    let depsFile = File.ReadAllText (gitDirectory </> "paket.dependencies")
    let replacedFile =
        depsFile
            .Replace("// FAKE_MYGET_FEED (don't edit this line)", "source https://www.myget.org/F/fake-vsts/api/v3/index.json")
            .Replace("prerelease // FAKE_VERSION (don't edit this line)", nugetVersion)
    File.WriteAllText(gitDirectory </> "paket.dependencies", replacedFile)
```

And I think `Replace` was marked the usage then is from `depsFile` to the next line until `(` which is longer than the last line